src/lib.js

```diff
@@ -165,6 +165,7 @@ export default class Migrator {
         this.log('There are no migrations to run'.yellow);
         this.log(`Current Migrations' Statuses: `);
         await this.list();
+        return []
```

Comment:
ah ok, got it. yes, i wondered if that was intended behaviour. i figured the Error had been put in for a reason. so you want to return [] for both cases? if so, i'll update the PR.